docs/recipes.rst

```diff
@@ -13,7 +13,8 @@ Recipes
 .. recipe-gallery::
     :layout: multi-row
     
-    recipes/scaffold_an_app images/recipes/scaffold_pic.png [scaffold, app, new_app]
+    recipes/scaffold_an_app_via_the_portal images/recipes/create_app_form.png [scaffold, app, new_app]
```

Comment:
I just merged a PR with lots of changes to the recipes page. You'll need to rebase.